library/core/src/error.md

```diff
@@ -17,7 +17,7 @@ The following are the primary interfaces of the panic system and the
 responsibilities they cover:
 
 * [`panic!`] and [`panic_any`] (Constructing, Propagated automatically)
-* [`PanicInfo`] (Reporting)
+* [`PanicHookInfo`] (Reporting)
```

Comment:
Should this list mention `PanicInfo` as well? This is in the core docs after all.